lib/tinybucket/model/branch.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Tinybucket
+  module Model
+    # Branch
+    #
+    # @see https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D/refs/branches
+    #   Branch resource - Bitbucket Cloud REST API document
+    #
+    # @!attribute [rw] links
+    #   @return [Hash]
+    # @!attribute [rw] type
+    #   @return [String]
+    # @!attribute [rw] name
+    #   @return [String]
+    # @!attribute [rw] repository
+    #   @return [Hash]
+    # @!attribute [rw] target
+    #   @return [Hash]
+    class Branch < Base
+      include Tinybucket::Model::Concerns::RepositoryKeys
+
+      acceptable_attributes :links, :type, :name, :repository, :target
+
+      def commits
+        commits_api.branch(name)
```

Comment:
This code should use `Commits Resource` to be enumerable, just like [Repository#commits](https://github.com/hirakiuc/tinybucket/blob/master/lib/tinybucket/model/repository.rb#L109).

But this pull request has already many changes. 
So it is better to keep this code right now, and fixed in another issue, I think. 😄